### PR TITLE
Add new metadata to download section 

### LINKED ISSUE #205: Add ontology terms to table of metadata in download section
In reviewing the docs for AnnData release, I noticed that we haven't yet updated the contents of the table that describes the metadata file included with downloads (see https://github.com/AlexsLemonade/scpca-docs/blob/main/docs/download_files.md#metadata). We now have additional columns for ontology term ids that should be accounted for in that file. 

The current contents of sample metadata can be found here: https://github.com/AlexsLemonade/ScPCA-admin/blob/main/sample_info/README.md#sample-data

### PR DESCRIPTION
**Pull request by allyhawkins**
Closes #205 

Here, I added some new entries to the table that describes the contents of the metadata file that is included with download. We are now including `organism` as well as the other ontology terms for each sample, so we should include that information. I mostly took what was in the `ScPCA-admin` readme about the new entries, but condensed it to be more relevant to external users. 

I left in the description of NA/unknown for each of the fields except the disease ontology. I don't know where we would be missing a disease ontology since all samples should have a disease... Let me know if the level of detail that I included is okay.

### CODE REVIEW

File: docs/download_files.md [hunk 1]
```diff
@@ -105,6 +105,13 @@ The `single_cell_metadata.tsv` file is a tab-separated table with one row per li
 | `subdiagnosis`    | Subcategory of diagnosis or mutation status (if applicable)    |
 | `tissue_location` | Where in the body the tumor sample was located                 |
 | `disease_timing`  | At what stage of disease the sample was obtained, either diagnosis or recurrence |
+| `organism`         | The organism the sample was obtained from (e.g., `Homo_sapiens`) |
+| `development_stage_ontology_term_id` | `HsapDv` ontology term indicating developmental stage. `unknown` indicates age is unavailable. |
```
Comment (jaclyn-taroni) [inline docs/download_files.md:109]: These are all going to correspond to terms that are specific to the age value, right? Might as well be more specific.

File: docs/download_files.md [hunk 2]
```diff
@@ -105,6 +105,13 @@ The `single_cell_metadata.tsv` file is a tab-separated table with one row per li
...
+| `sex_ontology_term_id`| [`PATO``](http://obofoundry.org/ontology/pato.html) term referring to the sex of the sample. `unknown` indicates sex is unavailable. |
```
Comment (jaclyn-taroni) [inline docs/download_files.md:110]:
```suggestion
| `sex_ontology_term_id`| [`PATO`](http://obofoundry.org/ontology/pato.html) term referring to the sex of the sample. `unknown` indicates sex is unavailable. |
```

File: docs/download_files.md (hunk 1 above)
Comment (jaclyn-taroni) [reply docs/download_files.md:109]: Link to ontology? http://obofoundry.org/ontology/hsapdv.html

File: docs/download_files.md [hunk 3]
```diff
@@ -105,6 +105,13 @@ The `single_cell_metadata.tsv` file is a tab-separated table with one row per li
...
+| `organism_ontology_id`| NCBI taxonomy term for organism, e.g. [`NCBITaxon:9606`](https://ontobee.org/ontology/NCBITaxon?iri=http://purl.obolibrary.org/obo/NCBITaxon_9606). |
+| `self_reported_ethnicity_ontology_term_id` | For _Homo sapiens_, a [`Hancestro` term](http://obofoundry.org/ontology/hancestro.html). `multiethnic` indicates more than one ethnicity is reported. `unknown` indicates unavailable ethnicity and `NA` is used for all other organisms. |
```
Comment (jaclyn-taroni) [inline docs/download_files.md:112]:
```suggestion
| `self_reported_ethnicity_ontology_term_id` | For _Homo sapiens_ samples, a [`Hancestro` term](http://obofoundry.org/ontology/hancestro.html). `multiethnic` indicates more than one ethnicity is reported. `unknown` indicates unavailable ethnicity and `NA` is used for all other organisms. |
```

**Review by jaclyn-taroni — COMMENTED** [review]
There is one typo to be fixed, and I think we could be more helpful regarding the `HsapDv` values.

**Comment by allyhawkins** [comment]
I added the link for `HsapDv` terms and also specified that the ontology term indicates the age. This is ready for another look @jaclyn-taroni.

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻

**Pull request merged**